test/alternator/util.py

```diff
@@ -136,9 +136,14 @@ def unique_table_name():
 
 def create_test_table(dynamodb, **kwargs):
     name = unique_table_name()
+    BillingMode = 'PAY_PER_REQUEST'
+    if 'BillingMode' in kwargs:
+        BillingMode = kwargs['BillingMode']
+        del kwargs['BillingMode']
+
     print("fixture creating new table {}".format(name))
     table = dynamodb.create_table(TableName=name,
-        BillingMode='PAY_PER_REQUEST', **kwargs)
+        BillingMode=BillingMode, **kwargs)
```

Comment:
Please note that PAY_PER_REQUEST should continue to be not just the default, but the option used in 99.9% of the test - because creating a PROVISIONED table on Amazon DynamoDB has the potential to be a very costly mistake!

So I'm not very happy of making it easier to mistakenly publish a test with PROVISIONED billing mode, but I guess I'll wait and see the next patch why you needed this change.

Reply:
I hope the test makes it clearer. By default, nothing will be changed; instead, the hard-coded value is now optional.